tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_introduction.md

```diff
@@ -37,13 +37,6 @@ Multi-physics Object Oriented Simulation Environment
 
 - 30M tests per week
 
-
-!---
-
-## MOOSE Team
-
-!media darcy_thermo_mech/moose_team.png style=width:85%;margin-left:auto;margin-right:auto;display:block;
-
```

Comment:
I've updated this figure to include all current and past members, so this slide should be left in.

Reply:
Resolved offline.